spec/rubocop/cop/rspec/verified_doubles_spec.rb

```diff
@@ -55,27 +55,27 @@
     end
   end
 
-  it 'doubles that have no name specified' do
-    expect_offense(<<-RUBY)
-      it do
-        foo = double
-              ^^^^^^ Prefer using verifying doubles over normal doubles.
```

Comment:
It's quite strange how it worked like that, since `IgnoreNameless: true` by default.
Nice, it's fixed now.

This gives me an interesting thought, since on the one hand, it's a bug fix, and on the other, it's a change in behaviour. But we're unable to "protect" our users from that change.

Reply:
Perhaps `cop_config` was not merged with the default values?
Anyway, it's better to have the values explicitly set in specs and not rely on defaults

Reply:
Exactly. Previously – before https://github.com/rubocop-hq/rubocop/pull/7970 – the default configuration was _not_ present when testing. And it is now, so we can simplify a lot of our test.

I was too working on fixing our specs (and applied the _exact_ same changes to spec/rubocop/cop/rspec/verified_doubles_spec.rb) and have taken the liberty of adding a few commits to your branch. I hope that is ok with you @pirj

Reply:
This is such a wonderful idea. I recall it was discussed before, and so good it's now in place.

> is ok with you

Of course, nice cleanup!

Reply:
I mention in the commit message of 4e7272b7e0eda702006b1022950cc111ca707295 that the change in RuboCop actually introduces a “mystery guest”, which could be considered a code smell.

Reply:
Everything works and looks nice. Do you have something else in mind tbd here, or is it good to merge?

Reply:
I think we‘re good to merge.